platform/golden/bullseye.pkr.hcl

```diff
@@ -53,16 +53,30 @@ variable "ssh_password" {
 }
 
 source "virtualbox-iso" "base-debian-amd64" {
+  # boot instructions/commands to send to grub/bootloader to pass preseed
   boot_command         = [
     "<esc><wait>",
-    "auto <wait>",
-    "fb=false <wait>",
-    "install <wait>", 
-    "passwd/username=packer <wait>",
-    "passwd/user-password=${var.ssh_password} <wait>",
-    "passwd/user-password-again=${var.ssh_password} <wait>",
-    "preseed/url=http://{{ .HTTPIP }}:{{ .HTTPPort }}/${var.config_file}<wait>",
-    "<enter><wait>"
+    "c <wait>",
+    "set default=0 <enter><wait>",
+    "linux /install.amd/vmlinuz ",
+    "url=http://{{ .HTTPIP }}:{{ .HTTPPort }}/${var.config_file} ",
+    "passwd/username=packer ",
+    "passwd/user-password=${var.ssh_password} ",
+    "passwd/user-password-again=${var.ssh_password} ",
+    "debian-installer/language=en debian-installer/country=US ",
+    "console-setup/ask_detect=false ",
+    "console-setup/layoutcode=us ",
+    "keyboard-configuration/layoutcode=us ",
+    "keyboard-configuration/xkb-keymap=us ",
+    "debian-installer/keymap=skip-config ",
+    "debian-installer/locale=en_US.UTF-8 ",
+    "localechooser/preferred-locale=en_US.UTF8 ",
+    "netcfg/get_hostname=golden ",
+    "netcfg/get_domain=local ",
+    "grub-installer/bootdev=/dev/sda ",
```

Comment:
I belive, but am not certain, grub-installer should install on `/dev/vda`, not `/dev/sda`.